yt_dlp/extractor/vimeo.py

```diff
@@ -121,7 +122,13 @@ def _parse_config(self, config, video_id):
         video_data = config['video']
         video_title = video_data.get('title')
         live_event = video_data.get('live_event') or {}
-        is_live = live_event.get('status') == 'started'
+        live_status = traverse_obj(live_event, ('status', {lambda status: {
+            'pending': 'is_upcoming',
+            'active': 'is_live',  # TODO: is live or not?
```

Comment:
Note: confirm before merge

Reply:
iirc this is the status of a vimeo event where it *is* streaming but the event itself has not actually started yet, so the stream is just promo/placeholder image(s)/footage and a countdown clock. Since we would be able to start downloading at this point, we should probably consider it `is_live`